pkg/runtime/runtime.go

```diff
@@ -3004,7 +3004,7 @@ func isEnvVarAllowed(key string) bool {
 	}
 
 	// If we have a `DAPR_ENV_KEYS` env var (which is added by the Dapr Injector in Kubernetes mode), use that as allowlist too
-	allowlist := os.Getenv(authConsts.EnvKeysEnvVar)
+	allowlist := strings.ToUpper(os.Getenv(authConsts.EnvKeysEnvVar))
```

Comment:
move this and its tests to another PR if possible with associated issue if any.

Reply:
Tests don't pass without this. It's a fixup needed or env vars that aren't all uppercase would not work on Linux.